.github/workflows/oci.yml

```diff
@@ -5,16 +5,20 @@
 # https://github.com/crazy-max/ghaction-docker-meta
 # https://github.com/docker/build-push-action
 # https://github.com/MarcelCoding/luna/blob/main/.github/workflows/ci.yaml
-name: Release
+name: OCI
 
 on:
-  schedule:
-    - cron: '0 10 * * *' # everyday at 10am
   push:
     tags:
       - '*.*.*'
+
+  # Allow job to be triggered manually.
   workflow_dispatch:
 
+  # Build nightlies.
+  schedule:
+    - cron: '0 5 * * *'
```

Comment:
Is it ok that it's on the same time with `builds.yml`?

Reply:
Yes it's okay. Both workflow jobs are independent from each other.